src/Tools/GenerateProps/targetgroups.props

```diff
@@ -74,6 +74,9 @@
     </TargetGroups>
     <TargetGroups Include="netstandard">
       <NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
+      <!-- By default, netstandard libraries will be held to the UWP compatibility bar;
+           set this to false in the project to force otherwise. -->
+      <EnableWinRT>true</EnableWinRT>
```

Comment:
@weshaggard remind me of the reason for this name -- I believe you mentioned use of WinRT outside the context of UWP? I don't think i see where this property would enable such use though. It does seem like it needs renaming.